collections/s5p-no2-tropno-daily-check.yaml

```diff
@@ -4,6 +4,8 @@ AdditionalInfoExternal:
     Title: Additional info
     Path: s5p-no2-tropno-daily-check/README.MD
 Image: s5p-no2-tropno-daily-check/s5p-no2-tropno-daily-check.png
+EDCBrowser: https://browser.eurodatacube.com/?zoom=3&lat=21.89208&lng=-48.28697&collectionId=sentinel-5p-l2-tropomi&layerId=Nitrogen%20Dioxide%20tropospheric%20column&type=sentinel-hub-edc&fromTime=2021-03-04T16%3A30%3A21.421Z&toTime=2022-03-04T16%3A30%3A21.421Z
+EOBrowser: https://sentinelshare.page.link/WLoV
```

Comment:
Are you sure it the same collection as S5P Nitrogen Dioxide in EO browser ?

Reply:
You're right. I linked to the wrong data. This data collection is not on EO Browser.